community/import-tool/src/test/java/org/neo4j/tooling/ImportToolTest.java

```diff
@@ -1095,6 +1110,23 @@ public boolean accept( int line )
         };
     }
 
+    private String executeImportAndCatchOutput(String... arguments)
+    {
+        PrintStream originalStream = System.out;
+        ByteArrayOutputStream outputStreamBuffer = new ByteArrayOutputStream();
+        PrintStream replacement = new PrintStream( outputStreamBuffer );
+        System.setOut( replacement );
```

Comment:
It looks quite strange that we need to replace `System.out` for this test.
Maybe it is better to invoke some other method rather than `ImportTool#main` that can take a `PrintStream`?